e2e-tests/cypress/integration/before-you-start-appellant-selects-the-site.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Appellant selects the site
```

Comment:
Why is the feature file in this subtask?

Reply:
rebasing issues